VATSpy.dat

```diff
@@ -19311,39 +19311,104 @@ LECP|Palma (FL245-)|LEPA|LECP
 LECS|Sevilla||LECS
 LECS-NCS|Sevilla (East) - Sevilla|LECS_NCS|LECS-NCS
 LECS-SM2|Sevilla (West) - Sevilla|LECS_SM2|LECS-SM2
-LFBB|Bordeaux||LFBB
-;LFBB-UN|Bordeaux|LFBB_UN|LFBB-UN
-LFBB-US|Bordeaux|LFBB_US|LFBB-US
-LFBB-H|Bordeaux|LFBB_H|LFBB-H
-LFBB-N|Bordeaux|LFBB_N|LFBB-N
-LFBB-Z|Bordeaux|LFBB_N|LFBB-Z
-LFBB-X|Bordeaux|LFBB_N|LFBB-X
-LFBB-P|Bordeaux|LFBB_P|LFBB-P
-LFBB-R|Bordeaux|LFBB_R|LFBB-R
-LFBB-L|Bordeaux|LFBB_L|LFBB-L
-LFBB-T|Bordeaux|LFBB_T|LFBB-T
-LFEE|Reims (Covers North-East LFFF FL295+)||LFEE
-LFEE-N|Reims North (Covers North-East LFFF FL295+)|LFEE_N|LFEE-N
-LFEE-S|Reims South|LFEE_S|LFEE-S
+LFBB|Bordeaux|LFBB|LFBB
+LFBB-H|Bordeaux ACC (FL195+) - Bordeaux|LFBB_H|LFBB-H
+LFBB-L|Bordeaux ACC (FL195+) - Bordeaux|LFBB_L|LFBB-L
+LFBB-N|Bordeaux ACC (FL195+) - Bordeaux|LFBB_N|LFBB-N
+LFBB-P|Bordeaux ACC (FL195+) - Bordeaux|LFBB_P|LFBB-P
+LFBB-R|Bordeaux ACC (FL195+) - Bordeaux|LFBB_R|LFBB-R
+LFBB-T|Bordeaux ACC (FL195+) - Bordeaux|LFBB_T|LFBB-T
+LFBB-UN|Bordeaux ACC (FL195+) - Bordeaux|LFBB_UN|LFBB-UN
+LFBB-US|Bordeaux ACC (FL195+) - Bordeaux|LFBB_US|LFBB-US
+LFBB-X|Bordeaux ACC (FL195+) - Bordeaux|LFBB_X|LFBB-X
+LFBB-Z|Bordeaux ACC (FL195+) - Bordeaux|LFBB_Z|LFBB-Z
+LFEE|Reims|LFEE|LFEE
+LFEE-N|Reims ACC (FL265+) - Reims|LFEE_N|LFEE-N
+LFEE-E|Reims ACC (FL195+) - Reims|LFEE_HE|LFEE-E
+LFEE-E|Reims ACC (FL195+) - Reims|LFEE_HH|LFEE-E
+LFEE-E|Reims ACC (FL195+) - Reims|LFEE_KD|LFEE-E
+LFEE-E|Reims ACC (FL195+) - Reims|LFEE_KE|LFEE-E
+LFEE-E|Reims ACC (FL195+) - Reims|LFEE_KF|LFEE-E
+LFEE-E|Reims ACC (FL195+) - Reims|LFEE_KH|LFEE-E
+LFEE-E|Reims ACC (FL195+) - Reims|LFEE_UE|LFEE-E
+LFEE-E|Reims ACC (FL195+) - Reims|LFEE_UF|LFEE-E
+LFEE-E|Reims ACC (FL195+) - Reims|LFEE_UH|LFEE-E
+LFEE-E|Reims ACC (FL195+) - Reims|LFEE_XE|LFEE-E
+LFEE-E|Reims ACC (FL195+) - Reims|LFEE_XH|LFEE-E
+LFEE-N|Reims ACC (FL265+) - Reims|LFEE_HN|LFEE-N
+LFEE-N|Reims ACC (FL265+) - Reims|LFEE_HR|LFEE-N
+LFEE-N|Reims ACC (FL265+) - Reims|LFEE_KN|LFEE-N
+LFEE-N|Reims ACC (FL265+) - Reims|LFEE_KR|LFEE-N
+LFEE-N|Reims ACC (FL265+) - Reims|LFEE_UB|LFEE-N
+LFEE-N|Reims ACC (FL265+) - Reims|LFEE_UN|LFEE-N
+LFEE-N|Reims ACC (FL265+) - Reims|LFEE_UR|LFEE-N
+LFEE-N|Reims ACC (FL265+) - Reims|LFEE_XR|LFEE-N
+LFEE-N|Reims ACC (FL265+) - Reims|LFEE_YB|LFEE-N
+LFMM|Marseille|LFMM|LFMM
+LFMM|Marseille|LFMM_U|LFMM

```

Comment:
Not needed because any undefined LFMM sectors will default to main LFMM FIR boundary
```suggestion
```